packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -53,6 +54,9 @@ let workersConfigsWarningShown = false;
 
 let miniflare: Miniflare | undefined;
 
+/** The resolved inspector port (or undefined if inspecting is disabled) */
+let resolvedInspectorPort: number | undefined;
```

Comment:
Does this need to be in the module scope rather than the `cloudflare` function scope?

Reply:
ah, no you're right, sorry my bad 👍